includes/specials/SpecialUndelete.php

```diff
@@ -328,11 +328,16 @@ function isDeleted() {
 	function undelete( $timestamps, $comment = '', $fileVersions = array(), $unsuppress = false ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)